test/test_translate.py

```diff
@@ -99,12 +100,22 @@ class TestTranslateSpike(TestCase):
     """
     Tests for the translate.translateSpike function.
     """
-    def testNoGapsCorrectLength(self):
+    def testNoGapsCorrectSequence(self):
         """
-        A sequence with no gaps must have the correct length.
+        A sequence with no gaps must be translated correctly.
         """
         seq = 'TTGGTTGTTTATTACCAC'
-        self.assertEqual(len(seq) / 3, len(translateSpike(seq)))
+        self.assertEqual(Seq(seq).translate(), translateSpike(seq))
+
+    def testNoGapsCorrectSequenceNotMultipleOfThree(self):
+        """
+        A sequence with no gaps that is not a multiple of three must
+        raise an AssertionError.
+        """
+        seq = 'TTGGTTGTTTATTACCA'
+        error = (f'The lenght of a sequence to be translated must '
```

Comment:
Ideally this would be a more strict regex. Start with `^` and end with `$` and have the `.` escaped.

```python
error = (rf'^The .... \.$')
```